<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maze</class>
 <widget class="QWidget" name="Maze">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Maze</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>600</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBox">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#spinBox{
	background-color:rgb(193, 213, 255);
}</string>
           </property>
           <property name="maximum">
            <number>50</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1" colspan="2">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>43</width>
             <height>350</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="1" colspan="2">
          <widget class="QScrollBar" name="horizontalScrollBar">
           <property name="styleSheet">
            <string notr="true">#horizontalScrollBar{
	background-color:rgb(193, 213, 255);
}</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#pushButton{
	background-color:rgb(193, 213, 255);
}</string>
           </property>
           <property name="text">
            <string>创建迷宫</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="spinBox_2">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#spinBox_2{
	background-color:rgb(193, 213, 255);
}</string>
           </property>
           <property name="maximum">
            <number>80</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>43</width>
             <height>350</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" rowspan="8">
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>750</width>
             <height>600</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>300</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="MyWidget.qrc">:/images/images/maze.jpg</pixmap>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="1" colspan="2">
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#label{
	background-color:rgb(193, 213, 255);
}</string>
           </property>
           <property name="text">
            <string>速度调节</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pushButton_2">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#pushButton_2{
	background-color:rgb(193, 213, 255);
}</string>
           </property>
           <property name="text">
            <string>暂停</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="pushButton_3">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#pushButton_3{
	background-color:rgb(193, 213, 255);
}</string>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MyWidget.qrc"/>
  <include location="MyWidget.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Maze</receiver>
   <slot>CreateMaze()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>929</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar</sender>
   <signal>sliderReleased()</signal>
   <receiver>Maze</receiver>
   <slot>VChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>882</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>857</x>
     <y>670</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Maze</receiver>
   <slot>Control()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>835</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>702</x>
     <y>722</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Maze</receiver>
   <slot>MyExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>905</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>751</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CreateMaze()</slot>
  <slot>Control()</slot>
  <slot>VChange()</slot>
  <slot>MyExit()</slot>
 </slots>
</ui>
